olive/model.py

```diff
@@ -750,11 +811,11 @@ def __init__(
         version: Optional[int] = None,
     ):
         super().__init__(
+            framework=Framework.ONNX,
+            model_file_format=ModelFileFormat.ONNX,
```

Comment:
>model_file_format [](http://example.com/codeflow?start=12&length=17)

I am supposing the aml system doesn't support the composite model. right?

Reply:
AML does support composite model right now but it can be added later. since the composite model is just a list of ONNXModels, we should be able to reuse the logic for ONNXModels.

Reply:
*does not (sorry about the typo_